docs/getting-started.md

```diff
@@ -10,9 +10,7 @@ ScalarDL is scalable and practical Byzantine fault detection middleware for tran
 ScalarDL is composed of Ledger, Auditor, and Client SDK as shown in the following figure. ScalarDL Ledger manages application data in its own unique way using hash-chain and digital signature. ScalarDL Auditor is an optional component and manages a copy of Ledger data without depending on Ledger to identify the discrepancy between Ledger and Auditor data.
 The Client SDK is a set of user-facing programs to interact with Ledger and Auditor. For more details, please read the [design doc](design.md) and [implementation details](implementation.md).
 
-<p align="center">
-<img src="images/scalardl.png" width="480" />
-</p>
+![](images/scalardl.png)
```

Comment:
It's more like a question.
With this change, the site shows the original big image without centering, but is there any way to make it look like the same as before with markdown/jekyll?

Reply:
The CSS in the theme oddly doesn't have a selector for page images, so I adding one to a local build of the site. This style centers images within docs and shrinks them to a max width of 480px.

Below is a before-and-after screenshot of the page on the docs site:

![centered-shrunk-image-before-after](https://github.com/scalar-labs/scalardl/assets/23216828/e815f823-d613-4f2f-a732-5d5f7b6c71df)

I'll confirm this works as expected with some other cases (e.g., images smaller than 480px in width, long images, etc.) and implement it across the product docs sites.